react/__mocks__/render.js

```diff
@@ -10,9 +10,7 @@ export class Link extends Component {
   }
 
   render() {
-    return (
-      <a href="#">{this.props.children}</a>
-    )
+    return <a href="#">{this.props.children}</a>
```

Comment:
This implementation of the mock looks simpler and uses the page to add more data to the snapshot.
`export const Link = ({ page, children }) => <a href={page}>{children}</a>`